lib/fs/index.js

```diff
@@ -117,3 +116,15 @@ if (typeof fs.writev === 'function') {
     })
   }
 }
+
+// fs.realpath.native only available in Node v9.2+
+if (typeof fs.realpath.native === 'function') {
+  exports.realpath.native = u(fs.realpath.native)
+} else {
+  process.emitWarning(
+    'fs.realpath.native is not a function. ' +
+    'Ensure that fs-extra is imported before other dependencies that can change fs.\n\n' +
+    '\tsee https://github.com/jprichardson/node-fs-extra/pull/953',
```

Comment:
I don't think changing import order should be the suggested fix; flawed monkey-patches should be removed or fixed. I am also doubtful that linking to this lengthy PR will be useful. Suggested warning:

> fs.realpath.native is not a function. Is fs being monkey-patched?

Reply:
Good call. Rephrased as advised for clarity.